File: 4_single_inheritance.cpp
#include <bits/stdc++.h>
using namespace std;
class Company {
public:
    string name;
    int id;
    Company(string emp_name, int emp_id) {
        name = emp_name;
        id = emp_id;
    }
    void display() {
        cout << "Employee Name: " << name << " and Employee Id: " << id << endl;
    }
    
};
class Employee : public Company {
public:
    Employee(string emp_name, int emp_id) : Company(emp_name, emp_id) {}
};
int main(){

    Employee obj("Gray", 23143);
    obj.display();
    return 0;
}

File: 4_single_inheritance.py
class Company:
    def __init__(self,name,id) -> None:
        self.name = name
        self.id = id
    def display(self):
        print("Employee Name:", self.name ,"and" ,"Employee Id:", self.id )



class Employee(Company):
    def __init__(self, name,id,designation) -> None:
        super().__init__(name,id)
        self.designation = designation

    def display(self):
        super().display()
        print("Designation:", self.designation)
obj = Employee("Gray",23143,"Manager")
obj.display()